arch/arm/src/stm32/stm32_serial.c

```diff
@@ -465,6 +465,9 @@ struct up_dev_s
 #ifdef CONFIG_SERIAL_OFLOWCONTROL
   const uint32_t    cts_gpio;  /* U[S]ART CTS GPIO pin configuration */
 #endif
+#ifdef CONFIG_SERIAL_TIOCGICOUNT
```

Comment:
should we modify all stm32 chips ? I know it's a lot of work, but so far we've tried to synchronize changes between stm32. Or at least add a compilation error that this feature is not supported in other families but can be easily  ported from stm32

Reply:
@raiden00pl my plane it to add support for other stm32 families too, but for now only stm32. If some application try to use this IOCTL it will receive the error indicating it is not implemented.

Reply:
could you create issue for this work, so we don't forget ? :)

Reply:
Done